src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$size: 100px;
+$calendar-gap: 1px;
+$calendar-width: ($size + $calendar-gap) * 7;
+$calendar-padding: 10px;
+$font-size: 30px;
+$field-border: 1px;
+$field-translate-Y: -20px;
```

Comment:
There is no need to create variables for values that you use only once

Reply:
Understood